viewsb/backends/phywhisperer.py

```diff
@@ -0,0 +1,169 @@
+"""
+PhyWhisperer-USB backend for ViewSB
+Inspired by OpenVizsla backend
+
+
+This file is part of ViewSB
+"""
+
+# pylint: disable=maybe-no-member,access-member-before-definition
+
+import sys
+import errno
+import argparse
+import time
+from datetime import datetime
+
+from ..backend import ViewSBBackend
+from ..packet import USBPacket, USBControlTransfer, USBPacketID
+
+
+try:
+    from phywhisperer import USBEventSink, usb as pw
+
+    class ViewSBEventSink(USBEventSink):
+        """ PhyWhisperer USB event sink that submits packets for decoding. """
+
+
+        def __init__(self, backend, suppress_packet_callback=None):
+
+            # Store a reference to our parent backend, so we can submit USB data via it.
+            self.backend = backend
+
+            # Mark ourselves as having no packet pending.
```

Comment:
`pending_packet` doesn't seem to be used, here. (I think the same dead code exists in the OpenVizsla backend; which in turn was left over from the USBProxy backend.)

Reply:
removed